Client/Clique/Common/Models/Participant.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Participant.swift
+//  Clique
+//
+//  Created by Infinum on 14.12.2022..
+//
+
+import Foundation
+
+struct Participant: Codable {
+    let user: User
+    let status: Int
```

Comment:
No need to change stuff, but what does this `status` value mean? It's pretty unclear from the naming point of view.